packages-exp/installations-types-exp/index.d.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Provider } from '@firebase/component';
+import { FirebaseInstallationsInternal } from '../installations-exp/src/interfaces/installation-internal';
+
+export interface FirebaseInstallations {
+  /**
+   * Firebase APP configurations
+   */
+  readonly appConfig: AppConfig;
+  /**
+   * Firebase platform logging util.
+   */
+  readonly platformLoggerProvider: Provider<'platform-logger'>;
+}
+
+export interface AppConfig {
+  /**
+   * Firebase APP name.
+   */
+  readonly appName: string;
+  /**
+   * Firebase project ID.
+   */
+  readonly projectId: string;
+  /**
+   * Firebase API key.
+   */
+  readonly apiKey: string;
+  /**
+   * Firebase APP ID.
+   */
+  readonly appId: string;
+}
+
+export type FirebaseInstallationsName = 'installations-exp';
+
+declare module '@firebase/component' {
+  interface NameServiceMapping {
+    'installations-exp': FirebaseInstallationsInternal;
+  }
```

Comment:
Please move the definition of `FirebaseInstallationsInternal` here instead of importing it from the main package.

```suggestion
  interface NameServiceMapping {
    'installations-exp': FirebaseInstallations;
    'installations-internal-exp': FirebaseInstallationsInternal
  }
```

Reply:
done